lua/cscope_maps.lua

```diff
@@ -17,7 +17,7 @@ M.setup = function(opts)
 
 	if helper.is_inbuilt_cscope() then
 		if vim.loop.fs_stat(M.opts.cscope.db_file) ~= nil then
-			helper.init_inbuilt_cscope()
+			require("cscope").init_inbuilt_cscope(M.opts.cscope)
```

Comment:
This will skip rooter logic. Is that fine with you?

Reply:
Actually, I don't use your plugin on older(inbuilt cscope) versions of nvim ;)

I just happened to run nvim 0.7.2 from the Debian Bookworm repository to just check it was working on Linux too, after fixing a Windows path issue, then I noticed that it was outputting a red error message when the cscope.out file is exist.

So I just wanted to remove the error message with minimal code modifications, while making the pull request for other purposes.

But in conclusion, my commit today makes rooter work on ancient nvim as well. Cheers.